WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -0,0 +1,232 @@
+    /// Cells currently configured in the order they appear on screen
+    ///
+    func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
+        switch cell {
+        case let cell as TextFieldTableViewCell where row == .termTextField:
+            configureTextField(cell: cell)
+        case let cell as BasicTableViewCell where row == .selectedTerms:
+            configureOption(cell: cell)
+        case let cell as BasicTableViewCell where row == .existingTerms:
+            configureOption(cell: cell)
+        default:
+            fatalError()
+            break
+        }
+    }
+
+    func configureTextField(cell: TextFieldTableViewCell) {
+        let viewModel = TextFieldTableViewCell.ViewModel(text: nil,
+                                                         placeholder: Localization.titleCellPlaceholder,
+                                                         onTextChange: { newAttributeOption in
+
+            }, onTextDidBeginEditing: {
+        }, inputFormatter: nil, keyboardType: .default)
+        cell.configure(viewModel: viewModel)
+        cell.applyStyle(style: .body)
+    }
+
+    func configureOption(cell: BasicTableViewCell) {
+        cell.textLabel?.text = "Work in progress" //TODO: to be implemented
+    }
+}
+
+// MARK: - Keyboard management
+//
+private extension AddAttributeOptionsViewController {
+    /// Registers for all of the related Notifications
+    ///
+    func startListeningToNotifications() {
+        keyboardFrameObserver.startObservingKeyboardFrame()
+    }
+}
+
+extension AddAttributeOptionsViewController: KeyboardScrollable {
+    var scrollable: UIScrollView {
+        return tableView
+    }
+}
+
+
+// MARK: - Navigation actions handling
+//
+extension AddAttributeOptionsViewController {
+
+    @objc private func doneButtonPressed() {
+        // TODO: to be implemented
+    }
+}
+
+extension AddAttributeOptionsViewController {
+
+    struct Section: Equatable {
+        let header: String?
+        let footer: String?
+        let rows: [Row]
+    }
+
+    enum Row: CaseIterable {
+        case termTextField
+        case selectedTerms
+        case existingTerms
+
+        fileprivate var type: UITableViewCell.Type {
+            switch self {
+            case .termTextField:
+                return TextFieldTableViewCell.self
+            case .selectedTerms:
+                return BasicTableViewCell.self
+            case .existingTerms:
+                return BasicTableViewCell.self
+            }
+        }
+
+        fileprivate var reuseIdentifier: String {
+            return type.reuseIdentifier
+        }
+    }
+}
+
+private extension AddAttributeOptionsViewController {
+    enum Localization {
+        static let nextNavBarButton = NSLocalizedString("Next", comment: "Next nav bar button title in Add Product Attribute Options screen")
+        static let titleCellPlaceholder = NSLocalizedString("Option name",
```

Comment:
nit: maybe the name can be a bit more specific like `optionTitleCellPlaceholder` or `optionNameCellPlaceholder` (maybe this to be consistent with naming in `AddAttributeViewController`)? at first I thought it's for the name of the attribute :)